api-module-library/slack/api.js

```diff
@@ -144,7 +145,7 @@ class Api extends OAuth2Requester {
             url: this.baseUrl + this.URLs.lookupUserByEmail,
             body,
             headers: {
-                'Content-Type': 'application/json',
+                'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
What an odd thing to require by Slack. But, good catch!

Reply:
in postman it works even when content-type is json, but the docs say x-www-form-urlencoded